INSTALL.adoc

```diff
@@ -17,6 +17,8 @@
 
 - Install Python (3.6 or newer)
 
+- Install `npm` (If the version in your distribution is too old: `sudo npm install -g n && sudo n stable`)
```

Comment:
`sudo n`? Talán `npm` akart lenni, nem tudom.

Reply:
Még frissek az emlékek: konkrétan ezt írtam be és működött!

- https://www.npmjs.com/package/n

> n – Interactively Manage Your Node.js Versions
> Node.js version management: no subshells, no profile setup, no convoluted API, just simple.

Reply:
Ah, azt hittem typo. Akkor mehet :)